src/cli/commands/floodsub/start.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const utils = require('../../utils')
+const debug = require('debug')
+const log = debug('cli:floodsub')
+log.error = debug('cli:floodsub:error')
+
+module.exports = {
+  command: 'start',
```

Comment:
I agree with not having a start command. In an initial version I had it initialize when a call to `pub` or `sub` was made. My further comments on this are below :)